lib/client/connect.js

```diff
@@ -45,4 +49,11 @@ if(!browserId) {
 } else {
   // invoke the starting process
   KadiraDebug._init(browserId, clientId);
+}
+
+function getQueryParamFromURI(q) {
```

Comment:
Can't we use a npm module for that? Try to find a small npm module for this if it's there. Make sure, how much of code it's add to the app.